habitat/datasets/object_nav/object_nav_dataset.py

```diff
@@ -30,7 +30,26 @@ class ObjectNavDatasetV1(PointNavDatasetV1):
     content_scenes_path: str = "{data_path}/content/{scene}.json.gz"
 
     def to_json(self) -> str:
+        for ep in self.episodes:
+            ep.goals = str(ep.goals[0].object_id)
```

Comment:
Maybe better key can be `{ep.scene}_{ep.goals[0].category}`, then dataset with multiple scenes won't collide.